tensorboard/pip_package/build_pip_package.sh

```diff
@@ -110,6 +110,7 @@ build() (
   # env markers are handled (https://github.com/pypa/setuptools/pull/1081)
   export PYTHONWARNINGS=ignore:DEPRECATION  # suppress Python 2.7 deprecation spam
   pip install -qU wheel 'setuptools>=36.2.0'
+  pip install -r requirements.txt
```

Comment:
We shouldn’t need `grpc`, etc. just to build the Pip package, just as we
currently don’t need to install requirements even though TensorBoard
code depends on lots of other non-vendored packages. (And of course the
rest of TensorBoard already uses numpy.) Could you please define `data`
as a lazy module, like all the other nontrivial public exports, to avoid
this?

Reply:
The `pip install -r requirements.txt` appears to still be here.

Reply:
Sorry I missed that in the previous revision. Removed now.